src/msg/Message.h

```diff
@@ -234,6 +234,7 @@ class Message : public RefCountedObject {
   bi::list_member_hook<> dispatch_q;
 
 public:
+  void *libosd_context;
```

Comment:
a comment here would be helpful. also, can initialize inline now that we're using c++11 to avoid touching the `Message()` constructors
```c++
  void *libosd_context = nullptr; // completion context for messages from libosd
```